Description: Starts epmd as a daemon during test phase and kills it afterwards
Author: Sergei Golovan <sgolovan@nes.ru>
Reviewed-By: Anton Gladky <gladk@debian.org>
Bug-Debian: https://bugs.debian.org/819122
Last-Updated: 2016-05-13

--- a/Makefile
+++ b/Makefile
@@ -208,12 +208,13 @@ test_elixir: test_stdlib test_ex_unit te
 
 test_stdlib: compile
 	@ echo "==> elixir (exunit)"
-	$(Q) exec epmd & exit
+	$(Q) epmd -daemon
 	$(Q) if [ "$(OS)" = "Windows_NT" ]; then \
 		cd lib/elixir && cmd //C call ../../bin/elixir.bat -r "test/elixir/test_helper.exs" -pr "test/elixir/**/*_test.exs"; \
 	else \
 		cd lib/elixir && ../../bin/elixir -r "test/elixir/test_helper.exs" -pr "test/elixir/**/*_test.exs"; \
 	fi
+	$(Q) epmd -kill
 
 #==> Dialyzer tasks
 
